composer.json

```diff
@@ -68,6 +68,7 @@
         "webonyx/graphql-php": ">=0.13 <1.0"
     },
     "conflict": {
+        "doctrine/common": "<2.7",
```

Comment:
Why did you need this ?
We still require `^2.5.2` which could also be updated.

Reply:
This is `doctrine/common`, not `doctrine/orm`. See https://github.com/doctrine/common/pull/734

Reply:
My bad